test/ssl-tests/13-fragmentation.conf.in

```diff
@@ -184,4 +184,85 @@ our @tests = (
             MaxFragmentSize => 4 * 1024,
         }
     },
+    ####                          #######
+    # Default fragment size is 512.
+    # Default Application data size is 256.
+    {
+        name => "Maximum Fragment Len extension set to 512",
+        server => { },
+        client => {
+            extra => {
+                MaxFragmentLenExt => 512,
+            },
+        },
+        test => {
+            ApplicationData => 1024,
+            MaxFragmentSize => 1024,
+        }
+    },
+    {
+        name => "Maximum Fragment Len extension equal FragmentSize equal",
+        server => { },
+        client => {
+            extra => {
+                MaxFragmentLenExt => 2048,
+            },
+        },
+        test => {
+            ApplicationData => 2 * 1024,
+            MaxFragmentSize => 2 * 1024,
```

Comment:
I don't understand what you are trying to show by representing the ApplicationData and MaxFragmentSize values in this way?